sdk/cmake/config.cmake

```diff
@@ -1,5 +1,5 @@
 
-if(ARCH STREQUAL "i386")
+	if(ARCH STREQUAL "i386")
```

Comment:
Unexpected diff?

Reply:
Fixed. Will arrive next force-push. (I’m trying to keep my incremental changes from wasting time in CI.)